packages/core/src/tools/error.spec.ts

```diff
@@ -50,26 +65,94 @@ describe('formatUnknownError', () => {
       stack: [],
     }
 
-    const formatted = formatUnknownError(stack, undefined, 'Uncaught')
+    const formatted = computeRawError({
+      stackTrace: stack,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🥜 nitpick:** ‏renaming those `stack` variables could be nice as well
```suggestion
      stackTrace,
```